grid2op/tests/test_ma.py

```diff
@@ -0,0 +1,518 @@
+            
+            assert (ma_env._subgrids_cls[space][agent].sub_orig_ids[
+                        ma_env._subgrids_cls[space][agent].gen_to_subid]\
+                ==\
+                    ma_env._cent_env.gen_to_subid[
+                        ma_env._subgrids_cls[space][agent].mask_gen
+                    ]).all()
+            
+            assert (ma_env._subgrids_cls[space][agent].sub_orig_ids[
+                        ma_env._subgrids_cls[space][agent].storage_to_subid]\
+                ==\
+                    ma_env._cent_env.storage_to_subid[
+                        ma_env._subgrids_cls[space][agent].mask_storage
+                    ]).all()
+            
+            # This test didn't pass with the previous version
+            assert (ma_env._subgrids_cls[space][agent].sub_orig_ids[
+                        ma_env._subgrids_cls[space][agent].shunt_to_subid]\
+                ==\
+                    ma_env._cent_env.shunt_to_subid[
+                        ma_env._subgrids_cls[space][agent].mask_shunt
+                    ]).all()
+            
+            assert (ma_env._subgrids_cls[space][agent].sub_orig_ids[
+                        ma_env._subgrids_cls[space][agent].line_ex_to_subid]\
+                ==\
+                    ma_env._cent_env.line_ex_to_subid[
+                        ma_env._subgrids_cls[space][agent].mask_line_ex
+                    ]).all()
+            
+            assert (ma_env._subgrids_cls[space][agent].sub_orig_ids[
+                        ma_env._subgrids_cls[space][agent].line_or_to_subid]\
+                ==\
+                    ma_env._cent_env.line_or_to_subid[
+                        ma_env._subgrids_cls[space][agent].mask_line_or
+                    ]).all()
+            
+            assert (ma_env._subgrids_cls[space][agent].sub_orig_ids[
+                        ma_env._subgrids_cls[space][agent].interco_to_subid][
+                            ma_env._subgrids_cls[space][agent].interco_is_origin]\
+                ==\
+                    ma_env._cent_env.line_or_to_subid[
+                        ma_env._subgrids_cls[space][agent].mask_interco
+                    ][
+                        ma_env._subgrids_cls[space][agent].interco_is_origin
+                ]).all()
+            
+            assert (ma_env._subgrids_cls[space][agent].sub_orig_ids[
+                        ma_env._subgrids_cls[space][agent].interco_to_subid][
+                            ~ma_env._subgrids_cls[space][agent].interco_is_origin]\
+                ==\
+                    ma_env._cent_env.line_ex_to_subid[
+                        ma_env._subgrids_cls[space][agent].mask_interco
+                    ][
+                        ~ma_env._subgrids_cls[space][agent].interco_is_origin
+                ]).all()
+            
+        # We check that if 2 elements are on the same substation 
+        # in the subgrid, they are on the same substation 
+        # in the main grid (and vice-versa).
```

Comment:
You should really avoid copy pasting when you code... 

This comments is useless. It does "check that if 2 elements are on the same substation in the original grid, they are on the same substation in the corresponding subgrid"

Reply:
Changed it